NEWS.Rmd

```diff
@@ -86,6 +86,15 @@ corresponded to the simple (unweighted) median (#905).
 * The function hanged on particularly small datasets as it tried to fit too
 many components for the available data (#953).
 
+### `fit_DoseResponseCurve()`
+
+* For `mode = "interpolation"`, the object returned contains an additional
+`De.raw` column, which stores the value of the raw De value as computed by the
+fitting function. The only difference between the `De` and `De.raw` columns,
+is that the first sets the De to `NA` if negative, while the latter doesn't.
+This can be helpful when testing that the De values from a null are close to
+zero; for general applications, this column should be ignored (#957).
```

Comment:
I still don't understand this part. What can be tested that cannot be tested otherwise? Because the values "close to zero" will be still "close to zero". The negative values can be "close to zero" but this is coincidental. I don't think that this should be the argument here, because it does not make sense. Better would e to replace it by "It is then up to the user to decide what to do with those numbers (#957)". We can then also skip the "for general ..."